packages/frontend/core/src/components/pure/workspace-slider-bar/collections/collections-list.tsx

```diff
@@ -254,12 +251,20 @@ export const CollectionSidebarNavItemContent = ({
       {filtered.length > 0 ? (
         filtered.map(page => {
           return (
-            <Doc
-              docId={page.id}
-              parentId={dndId}
-              inAllowList={allowList.has(page.id)}
-              removeFromAllowList={removeFromAllowList}
+            <SidebarDocItem
               key={page.id}
+              docId={page.id}
+              postfixConfig={{
```

Comment:
better not pass object as props, it will cause frequent rerender

Reply:
Indeed, but it's too complex to customize so many behavior,  how about use `useMemo` or any suggestions?

Reply:
We will solve this in next refactor